docs/handler-api.mdx

```diff
@@ -1075,31 +1075,54 @@ NemirtingasEpicEmu.json edition from a handler example:
 ---
 ### 11. Nemirtingas Galaxy GoG Emulator
 
-Automatically set up Nemirtinga's Galaxy Emulator in Nucleus.
+Nemirtinga's Galaxy Emulator works by substituting the game's Galaxy64.dll file with its own custom dll.
+
+First, set up Nemirtinga's Galaxy Emulator in Nucleus.
+
+`Game.UseNemirtingasGalaxyEmu = true;   //Set to true to emulate GOG Galaxy`
+
+It is recommended that the emulator is used on the game without Nucleus once before scripting. This can be done by the following steps:
+
+1. Head to the root folder of NucleusCoop
+2. Open the utils folder and then open the NemirtingasGalaxyEmu folder. Copy the Galaxy64.dll file.
+3. Head to the installation folder of the game you wish to emulate.
+4. Find where Galaxy64.dll is stored. This shall be called **the Galaxy folder.**
+5. Rename Galaxy64.dll or move it from the Galaxy folder and paste the Galaxy64.dll from the NemirtingasGalaxyEmu folder.
+6. Locate and load the game's executable.
+
+This should generate a folder called ngalaxye_settings. This folder contains a .json file that keeps track of vital information. Keep track of where this folder is located as you will need it for the Nucleus script.
+
+7. If the game loads fine and the game does not crash when trying to use multiplayer, then the emulator is working. Feel free to remove the ngalaxye_settings folder and Galaxy64.dll, and restore the original Galaxy64.dll file to the Galaxy folder.
```

Comment:
This can be simplified a bit:
```suggestion
1. Head to `NucleusCo-op\utils\NemirtingasGalaxyEmu`
2. Copy the `Galaxy64.dll` file.
3. Head to the installation folder of the game you wish to emulate.
4. Find where Galaxy64.dll is stored. This shall be called **the Galaxy folder**.
5. Rename Galaxy64.dll or move it from the Galaxy folder and paste the Galaxy64.dll from the NemirtingasGalaxyEmu folder.
6. Locate and load the game's executable.

This should generate a folder called ngalaxye_settings. This folder contains a .json file that keeps track of vital information. Keep track of where this folder is located as you will need it for the Nucleus script.

7. If the game loads fine and the game does not crash when trying to use multiplayer, then the emulator is working. Feel free to remove the ngalaxye_settings folder and Galaxy64.dll, and restore the original Galaxy64.dll file to the Galaxy folder.
```